.sync.yml

```diff
@@ -20,12 +20,11 @@ Rakefile:
 spec/spec_helper.rb:
   mock_with: ':mocha'
   coverage_report: true
-.github/workflows/auto_release.yml:
-  unmanaged: false
+.github/workflows/ci.yml:
+  unmanaged: true
+.github/workflows/nightly.yml:
+  unmanaged: true
 .github/workflows/release.yml:
   unmanaged: false
-.github/workflows/pr_test.yml:
-  unmanaged: false
-.github/workflows/spec.yml:
+.github/workflows/release_prep.yml:
   unmanaged: false
```

Comment:
I wanted to suggest to enable the puppet-lint plugins parameter_documentation / parameter_types but I'm currently not sure if/how pdk-templates supports that or if they are enabled by default.

Reply:
As mentioned these checks are enabled by default 💯